Vostok.Applications.AspNetCore/Vostok.Applications.AspNetCore.csproj

```diff
@@ -23,6 +23,12 @@
     <OutputType>Library</OutputType>
     <IsPackable>true</IsPackable>
   </PropertyGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.1'">
+    <FrameworkReference Include="Microsoft.AspNetCore.App" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+    <PackageReference Include="Microsoft.AspNetCore" Version="2.2.0"/>
```

Comment:
.net core 2.2 & asp.net core 2.2 are not LTS releases so I think it's better to use 2.1 version here\
https://dotnet.microsoft.com/platform/support/policy/dotnet-core

Reply:
Also related to cement configurations names etc.